### DISCUSSION TITLE
Remix example not working?

### DISCUSSION THREAD

**Discussion by ryanwoodcox** [vanilla-extract-css/vanilla-extract#1338] [Q&A]
Is the Remix example working for anyone?
At best I get this error:
```
7:07:51 PM [vite] Pre-transform error: No CSS for file: /app/components/HelloWorld.css.ts
7:07:51 PM [vite] Pre-transform error: No CSS for file: /app/components/HelloWorld.css.ts (x2)
7:07:51 PM [vite] Error when evaluating SSR module /app/components/HelloWorld.tsx: failed to import "/app/components/HelloWorld.css.ts"
|- Error: No CSS for file: /app/components/HelloWorld.css.ts
```

I fixed the import to look like: `import { vitePlugin as remix } from '@remix-run/dev';`

The package.json looks like
```{
  "name": "vanilla-extract-example-remix",
  "version": "0.0.0",
  "private": true,
  "type": "module",
  "scripts": {
    "dev": "remix vite:dev",
    "build": "remix vite:build",
    "start": "remix-serve ./build/server/index.js"
  },
  "dependencies": {
    "@remix-run/node": "^2.6.0",
    "@remix-run/react": "^2.6.0",
    "@remix-run/serve": "^2.6.0",
    "@vanilla-extract/css": "^1.14.1",
    "isbot": "^4",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@remix-run/dev": "^2.6.0",
    "@types/react": "^18.2.55",
    "@vanilla-extract/vite-plugin": "^4.0.4",
    "vite": "^5.0.11"
  }
}```

**Comment by askoufis** [comment]
The remix vite plugin was only recently made stable, but the example has existed since before then, so the import is now out-of-date if you have the latest version of `@remix-run/dev`. We should get this fixed.

PRs welcome :)

**Reply by ryanwoodcox** [reply]
I'm testing this example because I'm experiencing the same error for my project and I wanted to get a sanity check.

After fixing the import manually I still am receiving the error described above.

Downgrading the @vanilla-extract/vite-plugin to version `3.9.5` got rid of that error and threw a new ` Error: Cannot find package 'lightningcss' imported...` error.

After installing the missing `lightningcss` dev dependency, the example is working fine!

Other than the example needing some love, this still feels like an issue with version 4+ of the `@vanilla-extract/vite-plugin`?

**Reply by askoufis** [reply]
Oh, I was a bit vague. You'll also need to update `@remix-run/dev` to the latest version in order to actually get the new plugin version since it'll be locked to an older version in the lockfile. I just tested this and it works fine, no need to downgrade the VE plugin.

**Reply by ryanwoodcox** [reply]
I've upgraded @remix-run/dev to `2.8.0`. I am still getting the error: `[vite] Internal server error: No CSS for file: /app/components/HelloWorld.css.ts`

My package.json
```
{
  "name": "vanilla-extract-example-remix",
  "version": "0.0.0",
  "private": true,
  "type": "module",
  "scripts": {
    "dev": "remix vite:dev",
    "build": "remix vite:build",
    "start": "remix-serve ./build/server/index.js"
  },
  "dependencies": {
    "@remix-run/node": "^2.6.0",
    "@remix-run/react": "^2.6.0",
    "@remix-run/serve": "^2.6.0",
    "@vanilla-extract/css": "^1.14.1",
    "isbot": "^4",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@remix-run/dev": "^2.8.0",
    "@types/react": "^18.2.55",
    "@vanilla-extract/vite-plugin": "^4.0.4",
    "lightningcss": "^1.24.0",
    "vite": "^5.0.11"
  }
}
```

What is the package.json that you have working for you?

**Reply by ryanwoodcox** [reply]
Same error if I upgrade @remix-run/node, @remix-run/react @remix-run/serve all to `2.8.0`.

I am on Windows. Let me try this on my macbook

**Reply by ryanwoodcox** [reply]
My macbook was able to run this example successfully using `@vanilla-extract/vite-plugin`  version: `4.0.4

Are you able to reproduce the issue with your team on Windows?

**Reply by ryanwoodcox** [reply]
Perhaps this is related to:
https://github.com/vanilla-extract-css/vanilla-extract/issues/1321

**Reply by askoufis** [reply]
I'll have to reproduce it on my windows machine. Would appreciate it if you could raise an issue for tracking purposes with a minimal reproduction. It's probably fine to just say `run the remix example on windows` for the reproduction since it's already quite minimal.

**Reply by ryanwoodcox** [reply]
Of course, will do that now

**Reply by ryanwoodcox** [reply]
Issue opened. Thanks for the input!

https://github.com/vanilla-extract-css/vanilla-extract/issues/1339

**Comment by ryanwoodcox** [comment]
Answered and fix confirmed via https://github.com/vanilla-extract-css/vanilla-extract/pull/1346

Thanks!
